app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -289,10 +289,13 @@ private void sendForms(FormRecord[] records) {
                         if (CommCareApplication.instance().isStorageAvailable()) {
                             //If storage is available generally, this is a bug in the app design
                             Logger.log(AndroidLogger.TYPE_ERROR_DESIGN, "Removing form record because file was missing|" + getExceptionText(e));

```

Comment:
@wpride Once your new logging PR is merged, can we change this log tag to be the new FORM_SUBMISSION one? It would be really helpful to have these all show up under 1 filter.